hangman.js

```diff
@@ -1 +1,107 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+var word = prompt('gimme a word:');
+var guesses = [];
+var attempts = word.length + 4;
+var attemptsLeft = attempts;
+var display = '';
+
+myl = 'a';
+place = word.indexOf(myl);
+
+function showDisplay() {
+  for(i = 0; i < word.length; i++) {
+    display += '-';
+  }
+  display;
+}
+function getGuess() {
+  if(display === word) {
+    attemptsLeft = 0;
+    win();
+  } else {
+    while (attemptsLeft > 0) {
+      var guess = prompt('You have ' + attemptsLeft + ' attempts. ' + 'guess a letter:\n' + display);
+      var place = word.indexOf(guess);
```

Comment:
One Issue I see here, is that you are grabbing the index value of the location of the string if it's a match, which is good! But what happens if the word contains multiple instances of that letter, for example: "happens", "guess", etc.. have two of the same letter. I think indexOf only returns the first location.